pkg/scheduler/framework/runtime/framework.go

```diff
@@ -1259,24 +1259,26 @@ func (f *frameworkImpl) SharedInformerFactory() informers.SharedInformerFactory
 	return f.informerFactory
 }
 
-func (f *frameworkImpl) pluginsNeeded(plugins *config.Plugins) map[string]config.Plugin {
-	pgMap := make(map[string]config.Plugin)
+func (f *frameworkImpl) pluginsNeeded(plugins *config.Plugins) sets.String {
+	pgMap := sets.String{}
```

Comment:
since this isn't a map anymore, maybe `pgSet`?

Reply:
sure. missed it.

Reply:
done. :)